.swiftformat

```diff
@@ -0,0 +1,40 @@
+# file options
```

Comment:
Let's use the configuration used on the other Swift repositories.

```
--binarygrouping none
--decimalgrouping none
--hexgrouping none
--indent 2 
--octalgrouping none 
--semicolons never 
--wraparguments before-first 
--wrapcollections before-first 
--wrapparameters before-first
--extensionacl on-declarations
--maxwidth 100
```

Reply:
I've updated the config